backend/sse/src/server.ts

```
@@ -0,0 +1,197 @@
+            return;
+        }
+        const user = await response.json();
+        console.log(user);
+        const clientId = req.params.clientId;
+        const userId = await redis.get(clientId);
+        console.log(`User ID: ${userId}, Client ID: ${clientId}`);
+        if (!userId) {
+            res.status(401).json({error: "Unauthorized"});
+            return;
+        }
+        if (userId !== null && parseInt(userId) !== user.data.id) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)